src/lib/components/top-nav.svelte

```diff
@@ -1,15 +1,30 @@
 <script lang="ts">
+  import { twMerge as merge } from 'tailwind-merge';
+
   import DarkModeMenu from '$lib/components/dark-mode-menu.svelte';
   import DataEncoderStatus from '$lib/components/data-encoder-status.svelte';
   import TimezoneSelect from '$lib/components/timezone-select.svelte';
   import { translate } from '$lib/i18n/translate';
 
   let screenWidth: number;
+
+  let className = '';
```

Comment:
You were using a type that was compatible with twmerge that wasn't just string, seems real nice